src/types.ts

```diff
@@ -40,6 +62,6 @@ export function isFrame(node: SceneNode): node is FrameNode {
   return node.type === 'FRAME'
 }
 
-export function isGroup(node: SceneNode): node is GroupNode {
+export function isGroup(node: BaseNode): node is GroupNode {
```

Comment:
What's the difference here though?

Reply:
To be transparent: I still have to dig in the differences and fully grasp the Figma types